apps/web/src/components/AssessmentStepTracker/utils.ts

```diff
@@ -342,16 +340,14 @@ export function transformPoolCandidateSearchInputToFormValues(
       : [],
     govEmployee: input?.isGovEmployee ? "true" : "",
     languageAbility: input?.applicantFilter?.languageAbility ?? "",
-    operationalRequirement:
+    operationalRequirements:
       input?.applicantFilter?.operationalRequirements?.filter(notEmpty) ?? [],
     pools: [poolId],
     priorityWeight: input?.priorityWeight?.map((pw) => String(pw)) ?? [],
     skills:
       input?.applicantFilter?.skills?.filter(notEmpty).map((s) => s.id) ?? [],
     workRegion:
       input?.applicantFilter?.locationPreferences?.filter(notEmpty) ?? [],
-    expiryStatus: CandidateExpiryFilter.Active, // add default filters
-    suspendedStatus: CandidateSuspendedFilter.Active,
```

Comment:
We need to keep these to prevent expired and suspended candidates from appearing on the list. My screenshot tool is glitching out :p but if you look closely you can see the expired date is true for the Perfect priority.

![image](https://github.com/user-attachments/assets/70212124-4353-486d-940c-622ef9f339eb)

I think you can set the `modifyFilterCount={-3}` and add those filters as well.

Reply:
okay, that makes sense. Added back in [add other filters back in](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11096/commits/e0fe70cf5f6fb6c00470bbc6548f9283cff9c3d8)

Reply:
Ya sorry I should of been more clear in the issue

Reply:
My bad Eric I meant to change the other function. I can push up the changes.